### PR TITLE
[Fluid] Linear-log wall law refactor

### PR DESCRIPTION
**Pull request by rubenzorrilla**
**📝 Description**
This PR refactors the linear-log law implementation in the old `MonolithicWallCondition` to be compatible with the current `NavierStokesWallCondition`. Current implementation has been checked against the old one by calculating the RHS and LHS with both conditions. Identical results are obtained.

Requires  #10571 to be merged.

### CODE REVIEW

**Comment by ddiezrod** [comment]
There is one trick you can use to make the review easier: you can use fluid/navier-stokes-wall-modelling as target branch here so you will only see the changes from this PR and when fluid/navier-stokes-wall-modelling is merged to master, github will automatically change here the target branch to master too. We do it like this in Gitlab and I think it is quite handy.

**Comment by rubenzorrilla** [comment]
> There is one trick you can use to make the review easier: you can use fluid/navier-stokes-wall-modelling as target branch here so you will only see the changes from this PR and when fluid/navier-stokes-wall-modelling is merged to master, github will automatically change here the target branch to master too. We do it like this in Gitlab and I think it is quite handy.

Nice. I'll try the next time. Thanks for the hint :wink:.

File: applications/FluidDynamicsApplication/custom_conditions/wall_laws/linear_log_wall_law.h [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
```
Comment (sunethwarna) [inline applications/FluidDynamicsApplication/custom_conditions/wall_laws/linear_log_wall_law.h]:
```suggestion
```

I would remove the empty groups.

File: applications/FluidDynamicsApplication/custom_conditions/wall_laws/linear_log_wall_law.h [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+    /// Destructor.
+    ~LinearLogWallLaw() = default;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
```
Comment (sunethwarna) [inline applications/FluidDynamicsApplication/custom_conditions/wall_laws/linear_log_wall_law.h]:
```suggestion
```

Removing empty groups

**Review by sunethwarna — DISMISSED** [review]
Thanks @rubenzorrilla :D

**Review by sunethwarna — APPROVED** [review]

**Pull request merged**